pkgs/by-name/aw/await/package.nix

```
@@ -27,6 +30,10 @@ stdenv.mkDerivation rec {
     install -Dm755 await -t $out/bin
     install -Dm444 LICENSE -t $out/share/licenses/await
     install -Dm444 README.md -t $out/share/doc/await
+    installShellCompletion --cmd await \
+      --bash <($out/bin/await --autocomplete-bash) \
+      --fish <($out/bin/await --autocomplete-fish) \
+      --zsh <($out/bin/await --autocomplete-zsh)
```

```suggestion
    installShellCompletion --cmd await autocomplete.{bash,fish,zsh}
```

Using the pregenerated files is still preferred, just need to set the cmd flag.